eve/workers/pod-builder/pod.yml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: worker
+spec:
+  containers:
+    - name: worker
```

Comment:
in this container your "workspace" is on `/home/eve/workspace` but I don't see any volume attached to it, so you're writing new files on the overlay. I would suggest that you create a new volume of type `emptyDir` just like the `docker-storage` and attach to it the path `/home/eve/workspace`

Reply:
this will increase the performance :)